logins-api/android/LoginsApi/loginsapi/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">LoginsApi</string>
```

Comment:
It appears this string isn't used? So maybe the entire file isn't?

Reply:
Yeah, we shouldn't be including resources at all for the library.